README.md

```diff
@@ -82,9 +82,18 @@ If you're using [local setup (dockerized testing node)](https://github.com/matte
 
 ### Testing
 
-At the moment, we don't have any tests, but we are working on it.
+At the moment, we are working on CLI tests.
 In the meantime, you can test the code manually by running the code in [development mode](#run-in-development-mode).
 
+### CLI tests
+
+To run CLI tests for zksync-cli do the following:
+1. Go to [test](`./test`) folder
+2. Run `npm i` and install zksync-cli `npm i zksync-cli`
+3. Make sure you have [Docker](https://docs.docker.com/engine/install/) on your system.
+4. Make sure you have `.env` file with your wallet private key in `./tests/src/` dir. Key is `E2E_TESTNET_PK`. It may looks like `E2E_TESTNET_PK=012345...abcdef`
```

Comment:
`./tests/src/` - path is incorrect here.

Reply:
thanks, updated